.github/workflows/deploy.yaml

```diff
@@ -59,6 +59,17 @@ jobs:
       - name: push image
         run: docker push --all-tags $DOCKER_IMAGE
 
+      - name: Apply new schema
+        if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/staging'
+        run: |
+          # cloud run jobs to update
+          run_job_names=("$DEPLOY_TARGET-schema-migrator" "$DEPLOY_TARGET-reference-schema-migrator" "$DEPLOY_TARGET-reference-updater" "$DEPLOY_TARGET-reference-update-hpo-terms" "$DEPLOY_TARGET-reference-import-panels")
+          for cloud_run_target in run_job_names; do
```

Comment:
Just double-checking that it's okay to run these updates concurrently? I.e. there will never be any interdependencies or tables that are written to by the same jobs?

Reply:
These don't get run at the same time, this just updates the image dependency. I've split out the auto reference updaters, and schema is applied separately (because it's got the `--wait` on it).

Reply:
Thanks very much for explaining!